haaska.py

```diff
@@ -328,6 +391,29 @@ def set_percentage(self, val):
         vol = val / 100.0
         self._call_service('media_player/volume_set', {'volume_level': vol})
 
+class ClimateEntity(Entity):
+    def turn_on(self):
+        self._call_service('climate/set_away_mode', {'away_mode': False})
+
+    def turn_off(self):
+        self._call_service('climate/set_away_mode', {'away_mode': True})
+
+    def get_state(self):
+        state = self.ha.get('states/' + self.entity_id)
+        target_temp = state['attributes']['temperature']
+        heating_mode = 'AUTO'
+        if state['attributes']['away_mode'] == 'on':
+            heating_mode = 'AWAY'
```

Comment:
Yeah I was wondering about that. 
You can have some deviceMode "AWAY" (https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/smart-home-skill-api-reference#notsupportedincurrentmodeerror)
It does not seem to matter anyway, since the value is actually not used at all. But we maybe should keep it according to the docs.